ui/packages/catalog-ui-search/src/main/webapp/react-component/container/sharing/sharing.tsx

```diff
@@ -17,10 +17,15 @@ import { Restrictions, Access, Security, Entry } from '../../utils/security'
 const user = require('component/singletons/user-instance')
 const common = require('js/Common')
 const announcement = require('component/announcement')
-const store = require('../../../js/store.js')
+
+type Attribute = {
+  attribute: string
+  values: string[]
+}
 
 type Props = {
   id: number
+  updateMetacard: (attributes: Attribute[]) => void
```

Comment:
If we make this callback optional then the downstream projects depending on Sharing won't have to make any changes when this is merged.

I would consider renaming this to `onChange` or `onUpdate` as that's a bit more general.